pkg/api/handlers/generic/types.go

```diff
@@ -0,0 +1,55 @@
+package generic
+
+import (
+	"time"
+
+	docker "github.com/docker/docker/api/types"
+)
+
+// CPUStats aggregates and wraps all CPU related info of container
+type CPUStats struct {
+	// CPU Usage. Linux and Windows.
+	CPUUsage docker.CPUUsage `json:"cpu_usage"`
+
+	// System Usage. Linux only.
+	SystemUsage uint64 `json:"system_cpu_usage,omitempty"`
+
+	// Online CPUs. Linux only.
+	OnlineCPUs uint32 `json:"online_cpus,omitempty"`
+
+	// Usage of CPU in %. Linux only.
+	CPU float64 `json:"cpu"`
```

Comment:
I don't really like the name here. Maybe `TotalCPU`?

Reply:
`TotalCPUPercent` alternatively?

Reply:
I went with the same name that is used in `libpod/stats_config.go` for the same thing. (I assign it from that structure from field `CPU`). But surely, I can change it.